### ISSUE TITLE
Add Tabernacle and Author Disambiguator article page to /author/QXX/curation

### ISSUE THREAD

**Issue by lubianat** [fnielsen/scholia#1395]
Hello, 

A quick feature suggestion for the /curation page would be to include:

* Tabernacle links for adding new topics
* A link to Author Disambiguator's page based on articles (and not only on authors) 

This shiny app for quick contributions (very experimental) links to these pages: https://lubianat.shinyapps.io/quicklit/

Example: 

https://tabernacle.toolforge.org/?#/tab/manual/Q104137842/P921%3BP4510

and 

https://author-disambiguator.toolforge.org/work_item_oauth.php?id=Q104137842&batch_id=&match=1&author_list_id=&doit=Get+author+links+for+work

**Comment by Daniel-Mietchen** [comment]
Yes, including TABernacle links is a good idea, but if we do that, we should probably add a few more properties, e.g.
https://tabernacle.toolforge.org/?#/tab/manual/Q104137842/P921%3BP4510%3BP1433%3BP407%3BP953%3BP275%3BP2860%3BP577 .
